examples/lead-agent/lead-score-flow/main.py

```
@@ -0,0 +1,324 @@
+            score = asyncio.run(
+                score_single_candidate_async(candidate, job_description, additional_instructions)
+            )
+        
+        return {
+            "success": True,
+            "candidate_id": score.id,
+            "candidate_name": name,
+            "score": score.score,
+            "reason": score.reason,
+            "job_description": job_description or JOB_DESCRIPTION
+        }
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)